src/pipx/colors.py

```diff
@@ -1,11 +1,13 @@
 import sys
 from typing import Callable
 
-import colorama  # type: ignore
+from pipx.constants import WINDOWS
```

Comment:
Instead of getting the `WINDOWS` constant, I would just do the following:
```python
try:
    import colorama
except ImportError:
    colorama = None
```

Then just change `if PRINT_COLOR:` in the original code to `if PRINT_COLOR and colorama:`

That way we only have one platform check on whether or not colorama is installed in `setup.py`, and we always use colorama if it is installed.  And we don't have to worry about the possibility of our `WINDOWS` constant somehow being out of sync with `sys_platform="win32"`